Tests/RunnerPart2/Runner2Test.php

```diff
@@ -250,6 +252,184 @@ public function testComponentRunsIfSandboxProjectDoesNotExist(): void
         self::assertNotContains('Environment "AZURE_STORAGE_CONNECTION_STRING"', $containerOutput);
     }
 
+    public function testArtifactsUpload()
```

Comment:
Bacha testy koncej warningem ale pipeline projde. Tenhle test se realne celej neprovede.

```
There was 1 warning:

1) Keboola\DockerBundle\Tests\RunnerPart2\Runner2Test::testArtifactsUpload
Trying to configure method "listFiles" which cannot be configured because it does not exist, has not been specified, is final, or is static

phpvfscomposer:///code/vendor/phpunit/phpunit/phpunit:97
```

https://dev.azure.com/keboola-dev/docker-runner/_build/results?buildId=22966&view=logs&j=6996b571-32f4-5682-df84-57163fb00ec3&t=e76f1229-b889-5d8d-f267-f831e7c879fc&l=471